code/modules/uplink/uplink_item.dm

```diff
@@ -1112,6 +1112,17 @@ var/list/uplink_items = list() // Global list so we only initialize this once.
 
 	return C
 
+/datum/uplink_item/badass/annoydrone
+	name = "Annoyance drone"
+	desc = "A drone designed to create minor annoyances, and agitate the crew.  Will attempt to not interfere with your work."
+	cost = 1
+	item = /obj/item/drone_shell/annoyingdrone
+
+/datum/uplink_item/badass/annoydrone/spawn_item(turf/loc, obj/item/device/uplink/U, mob/user)
```

Comment:
I looked at several possible solutions to this, and while clunky codewise, this one seemed to be the best.

THe only other plausible solution would be to have using the drone shell bind it to the user, but that runs into the problem of a ghost entering too quickly, before the shell can be bound.